rusk-wallet/src/gql.rs

```diff
@@ -58,14 +61,50 @@ struct BlockResponse {
     pub block: Option<Block>,
 }
 
+// See `PhoenixTransactionEvent` for the reason for this struct
+// and allowing dead code here.
 #[serde_as]
-#[derive(Deserialize, Debug)]
-pub struct BlockData {
-    #[serde_as(as = "DisplayFromStr")]
-    pub gas_spent: u64,
-    pub sender: String,
+#[derive(Deserialize, Debug, Clone)]
+#[allow(dead_code)]
+struct Note {
+    stealth_address: StealthAddress,
+}
+
+// This struct is used instead of the one in `dusk_core::transfer`
+// because of the order-dependent deserialization bug in
+// the `dusk_core::transfer::phoenix::Note` https://github.com/dusk-network/phoenix/issues/274.
```

Comment:
this issue is marked as solved, does the bug still persist? If it's solved can we use the `PhoenixTransactionEvent` directly?

Reply:
A new version of the phoenix-core crate needs to be released and used first.